fw/rbcx-coprocessor/src/Esp32Manager.cpp

```diff
@@ -122,6 +123,8 @@ void Esp32Manager::onEnRising() {
     const bool currentEdge = pinRead(espEnPin);
     if (currentEdge && !m_previousEnEdge && m_enPinHolders == 0) {
         queueReset(m_lastDtr && !m_lastRts);
+    } else if (!currentEdge && m_previousEnEdge) {
+        softReset();
```

Comment:
Co ma toto delat?

Reply:
Zastavola to softReset uz kdyz zmacknes reset button, jinak by to bylo pri bootstrapu az kdyz ho pustis